hack/cherry_pick_pull.sh

```diff
@@ -131,12 +131,13 @@ function make-a-pr() {
   # when we shove the heredoc at hub directly, tickling the ioctl
   # crash.
   prtext="$(mktemp -t prtext.XXXX)" # cleaned in return_to_kansas
+  numandtitle=$(printf '%s\n' "${SUBJECTS[@]}")
```

Comment:
if this isn't used outside of this function, can we make this local?

Reply:
@cblecker good point! updated. Thanks!